### ISSUE TITLE
Accessibility updates

### ISSUE THREAD

**Issue by kjrawson** [CollegeOfTheHolyCross/dta#53]
Hi, Steven. We've done some accessibility testing and came across some minor changes that would be helpful to implement in order to make the site more accessible:

- The following features on the homepage need form labels: (1) the main search bar; and (2) the newsletter sign-up fields. Make sure this information is provided as a value, not placeholder.
- Need to embed value text for the pause/play button on the homepage carousel.
- Not all pages’ page titles appear in the browser bar (e.g., Topic, Genre, Latest), which makes it hard to navigate between tabs. We need to add page titles where missing and make sure they are consistent across the site.
- Form labels are missing on the Contact page, meaning some users may not be able to input their information.

Thanks!

**Comment by scande3** [comment]
For the main search bar, there is the following in the layout:
- `<label class="sr-only" for="search-field-header">Search DTA</label>`

In theory, that should be available if the user is utilizing a screen reader. I'll finish these as possible but wanted to know if this user was using a screen reader? If so, are they able to locate that label or is the code failing to display it as the "sr-only" class is supposed to work for? ("sr-only" refers to "screen reader only").

**Comment by kjrawson** [comment]
Here's what the grad student who flagged this said: "When I last checked this I was using an app to detect missing form labels, not a screen reader, so it sounds like that may have been a mistake on the app’s part, in which case we’re good!"
